### PR TITLE
NEXUS-4575 - Wait for async events after repo changes

### PR DESCRIPTION
**Pull request by velo**
Give repo save time to update and run async events

### CODE REVIEW

**Comment by bdemers** [comment]
+1

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus383/Nexus383SearchNGIT.java [hunk 1]
```diff
@@ -189,6 +189,7 @@ public class Nexus383SearchNGIT
 
         // Enabling default repo again
         getSearchMessageUtil().allowSearch( NEXUS_TEST_HARNESS_REPO, true );
+        getEventInspectorsUtil().waitForCalmPeriod();
```
Comment (peterlynch) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus383/Nexus383SearchNGIT.java:193]: a comment wouldn't hurt, especially if you are trying to fix some bug.

**Comment by mcculls** [comment]
+1

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus383/Nexus383SearchNGIT.java (hunk 1 above)
Comment (velo) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus383/Nexus383SearchNGIT.java:193]: Done

**Comment by peterlynch** [comment]
+1

**Pull request merged**